indico/modules/events/registration/api.py

```diff
@@ -76,6 +79,34 @@ def _process_PATCH(self):
         return self._get_result()
 
 
+class RHAPIRegistrants(RH):
+    """RESTful registrants API"""
+
+    @oauth.require_oauth('registrants')
+    def _checkParams(self):
+        event = ConferenceHolder().getById(request.view_args['event_id'], True)
+        if event is None:
+            raise NotFound("No such event")
+        if not event.canManageRegistration(request.oauth.user):
+            raise Forbidden()
+        self.event = event
+
+    def _process_GET(self):
+        registrants = self.event.getRegistrantsList()
+        registrant_list = []
+        for registrant in registrants:
+            registrant_entry = {
+                "registrant_id": registrant.getId(),
+                "checked_in": registrant.isCheckedIn(),
+                "full_name": registrant.getFullName(title=True, firstNameFirst=True),
+                "checkin_secret": registrant.getCheckInUUID(),
+            }
+            regForm = self.event.getRegistrationForm()
```

Comment:
why not move this up and add `personal_data` to the `registrant_entry = {...}` dict literal?